presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -896,7 +899,11 @@ public ScalarFunctionImplementation getScalarFunctionImplementation(Signature si
             return specializedScalarCache.getUnchecked(getSpecializedFunctionKey(signature));
         }
         catch (UncheckedExecutionException e) {
-            throw Throwables.propagate(e.getCause());
+            if (e.getCause() instanceof ByteCodeException) {
+                throw new PrestoException(StandardErrorCode.NOT_SUPPORTED, e.getCause());
```

Comment:
This will throw the original message, which has some problems:

* It mentions 255, but this is an internal detail which might be inaccurate for the user. For example, if we generate a class method, one of the parameters will be `this`.
* We can hit this condition in other cases such as generating projections, in which case the message is completely wrong.